src/main/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommand.java

```diff
@@ -6,17 +6,18 @@
 import org.slf4j.LoggerFactory;
 import ua.com.juja.microservices.keepers.slackbot.exception.WrongCommandFormatException;
 import ua.com.juja.microservices.keepers.slackbot.model.dto.UserDTO;
+import ua.com.juja.microservices.keepers.slackbot.utils.Utils;
 
-import java.util.*;
+import java.util.List;
 
 /**
  * @author Konstantin Sergey
  */
-@ToString(exclude = {"SLACK_NAME_PATTERN", "logger"})
+@ToString(exclude = {"slackNamePattern", "logger"})
 @EqualsAndHashCode
 public class SlackParsedCommand {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
-    private final String SLACK_NAME_PATTERN = "@([a-zA-z0-9\\.\\_\\-]){1,21}";
+    private String slackNamePattern;
```

Comment:
Why don't we use the same as another class:
`@Value("${keepers.slackNamePattern}")
 private String slackNamePattern;`

Reply:
It's because a class SlackParsedCommand is not under Spring maintenance, thus annotations will not work.

Reply:
I see, thanks!